packages/paste-website/src/pages/components/date-picker/index.mdx

```diff
@@ -0,0 +1,617 @@
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+import {Stack} from '@twilio-paste/stack';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import {useUID} from '@twilio-paste/uid-library';
+import {Combobox} from '@twilio-paste/combobox';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {
+  RequiredDatePicker,
+  DateRangePicker,
+  OnChangePicker,
+  HasErrorPicker,
+  DisabledPicker,
+  ReadOnlyPicker,
+  LabelOnlyPicker,
+  StackOfPickers,
+  MinAndMaxPicker,
+} from '../../../component-examples/DatePickerExamples.ts';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/date-picker"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/date-picker/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Date Picker"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/date-picker"
+  storybookUrl="/?path=/story/components-date-picker--default-date-picker"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<Callout>
+  <CalloutTitle>Using alpha components in your product</CalloutTitle>
+  <CalloutText>
+    Alpha components are ready for use in production and components like Date Picker won't likely see any API changes.
+    The term "Alpha" means there is only basic documentation at this time. Please share your experience using this Alpha
+    component by{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions/new" showExternal>
+      starting a discussion on GitHub
+    </Anchor>
+    .
+  </CalloutText>
+</Callout>
+
+## Guidelines
+
+### About Date Picker
+
+The Date Picker should be used when a date-only value is required from the user. It is an input field which accepts a value in <inlineCode>MM/DD/YYYY</inlineCode> format.
+
+Currently, the Date Picker is built on top of <Anchor href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date" showExternal>the HTML date picker</Anchor>, using an <inlineCode>input</inlineCode> field with <inlineCode>type="date"</inlineCode>, which comes with some inherent limitations.
+In order to release a functional and stable Date Picker for immediate usage, we decided to work with these limitations, with the intentions of updating this package to a custom-built Date Picker in a future release. The current API was designed to avoid releasing any breaking changes with future versions of the Date Picker.
```

Comment:
```suggestion
In order to release a functional and stable Date Picker for immediate use, we decided to work within these limitations with the intention of updating this package to a custom Date Picker in a future release. The current API was designed to avoid any breaking changes with future versions of the Date Picker.
```